be/src/storage/tablet_reader.cpp

```diff
@@ -357,33 +365,36 @@ Status TabletReader::_to_seek_tuple(const TabletSchema& tablet_schema, const Ola
         // CHAR type strings are truncated at the storage level after '\0'.
         if (f->type()->type() == OLAP_FIELD_TYPE_CHAR) {
             RETURN_IF_ERROR(datum_from_string(get_type_info(OLAP_FIELD_TYPE_VARCHAR).get(), &values.back(),
-                                              input.get_value(i), &_mempool));
+                                              input.get_value(i), mempool));
         } else {
-            RETURN_IF_ERROR(datum_from_string(f->type().get(), &values.back(), input.get_value(i), &_mempool));
+            RETURN_IF_ERROR(datum_from_string(f->type().get(), &values.back(), input.get_value(i), mempool));
         }
     }
     *tuple = SeekTuple(std::move(schema), std::move(values));
     return Status::OK();
 }
 
 // convert vector<OlapTuple> to vector<SeekRange>
-Status TabletReader::_parse_seek_range(const TabletReaderParams& read_params, std::vector<SeekRange>* ranges) {
-    if (read_params.start_key.empty()) {
+Status TabletReader::parse_seek_range(const TabletSharedPtr& tablet, const std::string& range_start_op,
+                                      const std::string& range_end_op, const std::vector<OlapTuple>& range_start_key,
+                                      const std::vector<OlapTuple>& range_end_key, std::vector<SeekRange>* ranges,
+                                      MemPool* mempool) {
+    if (range_start_key.empty()) {
         return {};
     }
 
-    bool inc_lower = read_params.range == "ge" || read_params.range == "eq";
```

Comment:
suggest to use {lower|upper}_bound_inclusive}, inc is common-seen name for increment